v2/pkg/scan/scan.go

```diff
@@ -151,8 +150,6 @@ var (
 
 // NewScanner creates a new full port scanner that scans all ports using SYN packets.
 func NewScanner(options *Options) (*Scanner, error) {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Intentional?

Reply:
yeah, it's deprecated with go 1.20 (`crypto/rand` should be used)